microshift_getting_started/microshift-architecture.adoc

```diff
@@ -2,30 +2,63 @@
 [id="microshift-architecture"]
 = Architecture
 include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
 :context: microshift-architecture
 
 toc::[]
 
-Learn the specifics of {product-title} architecture including design intent, how it differs from {OCP}, and API compatibility.
+Learn the specifics of {product-title} architecture including design intent, how it differs from {oke}, and API compatibility.
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. Because {product-title} is primarily as a platform for deploying applications, only the APIs and features essential to operating in edge and small form factor computing environments are included.
+
+For example, {product-title} is tasked with only the following Kubernetes cluster capabilities:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}. Virtual machines are handled by the operating system in {product-title} deployments.
+
+.{product-title} as part of {op-system-bundle}.
```

Comment:
The second attribute here, `{op-system-bundle}`, is not firing right in the HTML preview